docs/source/dataframe.rst

```diff
@@ -25,6 +25,7 @@ DataFrame Class and General APIs
 .. autosummary::
     :toctree: generated
     :nosignatures:
+    :template: class.rst
```

Comment:
I copied the `class.rst` and `fucntion.rst` from TorchData or some other domain library, but I actually don't know what style change they bring 😳

Reply:
me either ... but seems no harm to leave it here.